source/developers-guide/customer-streams-extension/index.md

```diff
@@ -0,0 +1,280 @@
+---
+layout: default
+title: Customer - Search and Streams
+github_link: developers-guide/customer-streams-extension/index.md
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: Customer - Search & Streams
+menu_order: 500
+---
+
+This document describes how to extend the customer stream and customer search part in shopware. In most cases, a plugin wants to provide more conditions or data for the search. This document describes, by way of a simple example, how this is possible.
+
+## Add own condition
+In order to extend the search with a further condition, the same procedure can be used as for the product search. The customer search is given a criteria object in which the different search conditions are summarized. First, it is necessary to define your own condition:
```

Comment:
Alternativer Vorschlag: Since the customer search is based on the same function as the product search, the way of extending the search is pretty similar. The customer search is given an criteria object in which the different search conditions are summarized. The first step is to define your own condition: